pkg/storage/storage.go

```diff
@@ -228,6 +228,11 @@ func CheckIsImageSignature(repoName string, manifestBlob []byte, reference strin
 		return true, NotationType, manifestContent.Subject.Digest, nil
 	}
 
+	// check cosign signature (OCI 1.1 support)
+	if manifestArtifactType == zcommon.ArtifactTypeCosign && manifestContent.Subject != nil {
```

Comment:
Should we also update gc's cleanReferrer ?

Reply:
fixed